packages/odyssey-react/src/components/Tag/Tag.module.scss

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.list {
```

Comment:
Why remove the `ods-` prefix?

Reply:
primarily because it isn't necessary but secondarily to keep the selectors terse and avoid the need for bracket notation within the importing component.